### ISSUE TITLE
High priority: robin_stocks 2.0.2 breaks all crypto trading

### ISSUE THREAD

**Issue by maconard** [jmfernandes/robin_stocks#254]
All of the crypto functions (`order_buy_crypto_by_price`, `order_buy_crypto_by_quantity`, etc) are just wrappers that call the underlying `order_crypto(...)` function (seen here: https://robin-stocks.readthedocs.io/en/latest/_modules/robin_stocks/robinhood/orders.html#order_crypto). 

This `order_crypto(...)` function is broken, as it seems to accidentally recursively call itself. As a result, this breaks *ALL* crypto ordering calls.

The source code is below:
```
@login_required
def order_crypto(symbol, side, quantityOrPrice, amountIn="quantity", limitPrice=None, timeInForce="gtc", jsonify=True):
    """Submits an order for a crypto."""
    try:
        symbol = symbol.upper().strip()
    except AttributeError as message:
        print(message, file=get_output())
        return None

    crypto_info = get_crypto_info(symbol)
    if crypto_info['display_only']:
        print("WARNING: The dictionary returned by crypto.get_crypto_info() for this crypto has key 'display_only' set to True. May not be able to trade this crypto.", file=get_output())

    orderType = "market"

    if side == "buy":
        priceType = "ask_price"
    else:
        priceType = "bid_price"

    if limitPrice:
        price = limitPrice
        orderType = "limit"
    else:
        price = round_price(get_crypto_quote_from_id(crypto_info['id'], info=priceType))

    if amountIn == "quantity":
        quantity = quantityOrPrice
    else:
        quantity = round_price(quantityOrPrice/price)

    payload = {
        'account_id': load_crypto_profile(info="id"),
        'currency_pair_id': crypto_info['id'],
        'price': price,
        'quantity': quantity,
        'ref_id': str(uuid4()),
        'side': side,
        'time_in_force': timeInForce,
        'type': orderType
    }

    url = order_crypto() # HERE *************************
    data = request_post(url, payload, json=True, jsonify_data=jsonify)

    return(data)
```
You can see the comment `# HERE *************************` highlighting the issue.

`robin_stocks` 1.7 was working fine, and I *just* updated to 2.0.2 and immediately found this issue, via a try/except giving: `order_crypto() missing 3 required positional arguments: 'symbol', 'side', and 'quantityOrPrice'`. This seems to be referencing this accidental recursive call, as the recursive call does not have any parameters. I presume this recursive call is meant to be a call to a function that gets the crypto order URL.

Thanks,

**Comment by maconard** [comment]
I found another issue reporting the same thing: #251

You can close whichever seems less useful.

**Comment by maconard** [comment]
@jmfernandes Hope you are well. You may want to take a look at this as soon as you can :)

**Comment by jmfernandes** [comment]
fixed in 2.0.3
